android-components/components/lib/crash/metrics.yaml

```diff
@@ -43,3 +43,78 @@ crash_metrics:
       - android-probes@mozilla.com
       - jnicol@mozilla.com
     expires: never
+
+crash:
+  uptime:
+    type: timespan
+    description: >
+      The application uptime. This is equivalent to the legacy crash ping's
+      `UptimeTS` field.
+    notification_emails:
+      - crash-reporting-wg@mozilla.org
+      - stability@mozilla.org
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1790569
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1790569#c12
+    data_sensitivity:
+      - technical
+    expires: never
```

Comment:
Since these never expires, please add the notification_emails: (Both android-probes@mozilla.com and the product manager's email).  This applies to below as well.

Reply:
Ignore my comment, didn't notice the emails were added above the `bugs:` section.

Reply:
`expires: never` still requires an individual contributor to be responsible for the collection, and their email noted here.